src/Fusion.Summary.Api/Fusion.Summary.Api.http

```diff
@@ -0,0 +1,6 @@
+@Fusion.Summary.Api_HostAddress = http://localhost:5229
+
+GET {{Fusion.Summary.Api_HostAddress}}/weatherforecast/
```

Comment:
What is this. Is this necessarry for this pr?

Reply:
Its a test, built in, came with the project. Removed it.